<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabBatchWidget</class>
 <widget class="QWidget" name="TabBatchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="MainLayout">
     <item>
      <widget class="QLineEdit" name="edit_filter_batch">
       <property name="placeholderText">
        <string>filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="button_add_batch">
       <property name="text">
        <string>Add batc&amp;h</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="BatchTableView" name="table_batch">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>25</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_add_batch" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BatchTableView</class>
   <extends>QTableView</extends>
   <header>BatchTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
